.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -0,0 +1,224 @@
+        fi
+
+    - uses: ./.github/actions/upload-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+        if-no-files-found: error
+
+  build-updater:
+    needs:
+    - parse-versions
+    - prepare-environment
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.4.2
+      volumes:
+      # The updater makes heavy use of /tmp files.
+      - /tmp:/tmp
+      - /usr:/mnt/usr
+      - /opt:/mnt/opt
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        include: ${{ fromJson(needs.parse-versions.outputs.versions) }}
+    env:
+      ROX_BUNDLE_VERSION: ${{ matrix.version }}
+      ROX_PRODUCT_TAG: ${{ matrix.tag }}
+    steps:
+    - name: Free up disk space
+      shell: bash
+      run: |
+        set +e
+        set -x
+        df -h
+        for delete in /usr/share/dotnet /usr/local/lib/android /opt/ghc /opt/hostedtoolcache/CodeQL; do
+          rm -rf "/mnt${delete:?}"
+        done
+        df -h
+
+    - name: Checkout specific reference
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: ${{ env.ROX_PRODUCT_TAG }}
+
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
+    - name: Build updater
+      run: |
+        echo "Building updater for version ${{ env.ROX_BUNDLE_VERSION }} on branch ${{ env.ROX_PRODUCT_TAG }}..."
+        make tag
+        make -C scanner bin/updater
+
+    - name: Download NVD
+      uses: ./.github/actions/download-artifact-with-retry
+      with:
+        name: nvd
+        path: nvd.zip
+
+    - name: Set Updater
+      run: |
+        chmod +x scanner/bin/updater
+        mkdir vulns
+
+    - name: Create Folder
+      run: mkdir -p definitions/${{ env.ROX_BUNDLE_VERSION }}
+
+    - name: Run Updater (single bundle)
+      env:
+        STACKROX_NVD_ZIP_PATH: nvd.zip
+      run: |
+        scanner/bin/updater export vulns --manual-url "${{ needs.prepare-environment.outputs.manual_url }}"
+        mv vulns/* definitions/${{ env.ROX_BUNDLE_VERSION }}/
+
+    - name: Run updater (multi bundle)
+      env:
+        STACKROX_NVD_ZIP_PATH: nvd.zip
+      run: |
+        scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" --split bundles
+        zip definitions/${{ env.ROX_BUNDLE_VERSION }}/vulnerabilities.zip bundles/*.json.zst
+
+    - name: Upload definitions artifacts
+      if: success() || failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: artifact_${{ env.ROX_BUNDLE_VERSION }}
+        path: definitions
+
+  upload-definitions:
+    needs:
+    - build-updater
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Download definitions artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: downloaded_artifacts
```

Comment:
You need `pattern` here, no?

Reply:
I am not sure I understand your comment here. Do you mean we need to have a pattern for the artifact path?

Reply:
Go to https://github.com/actions/download-artifact and check if the `pattern` input is required for your goal, also worth looking into `merge-multiple` to see if it fits your use case.

Reply:
OK, sounds good.